### DISCUSSION TITLE
Image gallery plugin

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/winter#909] [Q&A]
Hello,

I need image gallery, may be who know good plugin for winter, i find one (https://github.com/Hounddd/wn-lightgallery-plugin/issues/1)  but it not working. 
Other for octobercms very old and not working too.

**Comment by damsfx** [comment]
I am the developer of the plugin mentioned ... I use it on several projects without any problem.  
Please try to give more information about the error encountered in the issue created in the plugin repo.

**Comment by oim37** [comment]
Thank you for answer, i add more information.
